boston-housing/index.js

```diff
@@ -51,28 +51,56 @@ export const arraysToTensors = () => {
  *
  * @returns {tf.Sequential} The linear regression model.
  */
-export const linearRegressionModel = () => {
+export function linearRegressionModel() {
   const model = tf.sequential();
   model.add(tf.layers.dense({inputShape: [bostonData.numFeatures], units: 1}));
+
+  model.summary();
```

Comment:
Do we plan to show model summary in console? Can we show it in UI?

Reply:
For now it's okay. We'll switch to tf.vis when its model visualization component is ready.